internal/third_party/forked/golang/golang-lru/lru.go

```diff
@@ -104,11 +104,12 @@ func (c *Cache) RemoveOldest() {
 }
 
 func (c *Cache) removeElement(e *list.Element) {
```

Comment:
I think the fix needs to be in lru... Get is mutating since it moves things to the front:

```
 // Get looks up a key's value from the cache.
 func (c *Cache) Get(key Key) (value interface{}, ok bool) {
-       c.lock.RLock()
-       defer c.lock.RUnlock()
+       c.lock.Lock()
+       defer c.lock.Unlock()
        return c.cache.Get(key)
 }
```

Reply:
:facepalm: 

I didn't catch that... thanks so much! This was driving me crazy.